### PR TITLE
CON-3109 Add subject for ``Quest03/robots-harmony``

### PR DESCRIPTION
**Pull request by Oumaimafisaoui**
CON-3109

### CODE REVIEW

File: subjects/DPxAI/robots-harmony/README.md [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+
+#### 1- Create a New HTML File:
+
+Start by creating a new HTML file called `robot-harmony.html`.
+
+Set up the basic structure of the HTML document with a `<head>` and `<body>` inside an `<html>` tag.
+
+#### 2- Create a Container for the Robots:
+
+Inside the `<body>`, create a div element with the ID gallery.
```
Comment (lotoussa) [inline subjects/DPxAI/robots-harmony/README.md:57]: ID gallery could be in a format more visible ? like body in this sentence

File: subjects/DPxAI/robots-harmony/README.md [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 100px;
+  padding: 100px;
+  /* Change this color to reflect the color of your gallery */
+  background-color: #383086;
+}
+```
+
+- Add the following block of style to all you class's of `name-robot`. To do so, we follow the rule:
```
Comment (lotoussa) [inline subjects/DPxAI/robots-harmony/README.md:205]: typo here in the word "all 'you' class" ?

File: subjects/DPxAI/robots-harmony/README.md [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+
+**1- Code Editor:**
+
+First , remember that you will not be using the platform's code editor. Instead, you can use this online [tool](https://jsfiddle.net/) to create and modify your code.
+
+> Feel free to play with the settings and familiarize yourself with the environment!
+
+**2- How to submit your code:**
+
+------->Louis
```
Comment (lotoussa) [inline subjects/DPxAI/robots-harmony/README.md:33]: i will add the instruction tonight

**Review by lotoussa — COMMENTED** [review]
overall lgtm

File: subjects/DPxAI/robots-harmony/README.md [hunk 4]
```diff
@@ -0,0 +1,394 @@
...
+
+## Instructions
+
+Theses tasks are representing the mandatory part for this raid to be passed successfully. Bonus part is optional but highly encouraged for more fun!
+
+### Task 1: Set Up the Gallery Structure
+
+#### 1- Create a New HTML File:
+
+Start by creating a new HTML file called `robot-harmony.html`.
```
[the remark this answers is not in the document]
Comment (Oumaimafisaoui) [reply subjects/DPxAI/robots-harmony/README.md]: Fixed.

File: subjects/DPxAI/robots-harmony/README.md (hunk 1 above)
Comment (Oumaimafisaoui) [reply subjects/DPxAI/robots-harmony/README.md]: Fixed.

File: subjects/DPxAI/robots-harmony/README.md (hunk 2 above)
Comment (Oumaimafisaoui) [reply subjects/DPxAI/robots-harmony/README.md]: Fixed.

File: subjects/DPxAI/robots-harmony/README.md (hunk 3 above)
Comment (Oumaimafisaoui) [reply subjects/DPxAI/robots-harmony/README.md:33]: Fixed.

**Review by lotoussa — APPROVED** [review]

**Pull request merged**